projects/valtimo/components/src/lib/components/carbon-list/carbon-list.component.ts

```diff
@@ -154,6 +156,7 @@ export class CarbonListComponent<T> implements OnInit, AfterViewInit, OnDestroy
   @Input() showSelectionColumn = false;
   @Input() striped = false;
   @Input() hideToolbar = false;
+  @Input() translationKey = '';
```

Comment:
Change this to something more specific, like 'lockedTooltip'

Reply:
Changed